hamocc/mo_ifdefs.F90

```diff
@@ -0,0 +1,72 @@
+module mo_ifdefs
+
```

Comment:
Shouldn't this just go into `mo_control_bgc`? A separate module is not necessary in my opinion, particularly since we will eventually move away more or less completely from ifdefs and read the logical switches through the namelist(s).

Reply:
I agree. I have moved  all the new logic to to mo_control_bgc and removed mo_ifdefs.